### ISSUE TITLE
Header: Required ARIA attribute not present: aria-level

### ISSUE THREAD

**Issue by sophiafeng2017** [microsoft/fluentui#13454]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Describe the issue:

The TeachingBubble header doesn't have aria-level and ITeachingBubbleProps doesn't provide an attribute to customize.

### Please provide a reproduction of the issue in a codepen:

1. https://developer.microsoft.com/en-us/fluentui#/controls/web/teachingbubble
Open an example at Usage on the page.

2. Click "F12" and Inspect the header.

#### Actual behavior:

Title: WCAG 4.1.1,WCAG 4.1.2: Ensures elements with ARIA roles have all required ARIA attributes (.ms-TeachingBubble-headline)
Tags: Accessibility, WCAG 4.1.1, WCAG 4.1.2, aria-required-attr

Issue: Ensures elements with ARIA roles have all required ARIA attributes (aria-required-attr - https://dequeuniversity.com/rules/axe/3.5/aria-required-attr?application=msftAI)

Target application: Home - Fluent UI - https://developer.microsoft.com/en-us/fluentui#/controls/web/teachingbubble

Element path: .ms-TeachingBubble-headline

Snippet:  <p role="heading" class="ms-TeachingBubble-headline headline-412"

#### Expected behavior:

How to fix: 
Fix any of the following:
  Required ARIA attribute not present: aria-level

### Documentation describing expected behavior

 WCAG 4.1.1,WCAG 4.1.2: Ensures elements with ARIA roles have all required ARIA attributes

**Comment by jdhuntington** [comment]
Thanks for the report @sophiafeng2017 -- Some small discussion available on #13441 about workarounds or potential fixes. Would love to have a PR if you are interested.
